src/Comments.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useState, useEffect } from 'react';
 import Grid from '@mui/material/Grid';
 import CardComment from './CardComment';
-
+import Loader from './Loader';
```

Comment:
add a line break between imports and interface

Reply:
Done. Thanks.